ui/src/main/resources/Panels/Recent Files.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The panel should be located in the File Manager code space. Check the existing applications when in doubt (e.g. Blog application).